### ISSUE TITLE
Support for TPU?

### ISSUE THREAD

**Issue by martinlyons** [google/mediapipe#212]
As is the mediapipe tflite models can't be used by the Coral TPU because they're not fully quantized. I don't believe it's possible to do post-training integer quantization from an existing tflite model - you need the original model.

I know lots of people have asked for the original models (for finetuning) but this is another reason why they should be made available. I understand why the training data can't be provided but I'm at a loss why the models are made available in a format that can't be used on so many platforms (e.g. raspberry pi with a TPU, or Windows desktop with GPU), especially when one of those platforms - the TPU - is a Google hardware platform for inference using tensorflow lite!

Please reconsider - at present I think mediapipe is a good idea but so limited in its application that it will never have much interest.

**Comment by mgyong** [comment]
@martinlyons Thanks for the feedback. If you would like MediaPipe models to be available for different platforms, pls kindly file Github issue listing the different versions of the models you believe we should support and we will evaluate accordingly

Coincidently regarding EdgeTPU, we are working on getting MediaPipe examples working on Coral EdgeTPU and hopefully we will have something to share shortly.

**Comment by mcclanahoochie** [comment]
Hi,
Checkout the new [v0.6.5](https://github.com/google/mediapipe/releases/tag/v0.6.5) release for Coral Edge TPU support, with 2 examples.

Please send us an update on your amazing coral use case.

**Comment by 0x108** [comment]
@mcclanahoochie Hi! In order to use 3D pose estimation with depth (3D keypoints), I would have to use Mediapipe, if I'm right - looking through Posenet, I realized that 3D keypoints are not supported and only Mediapipe seems to. For this use-case I haven't found an edge TPU version - would you know if this would be possible? My use-case would be to use the Edge TPU for 3D pose estimation and feed the data into a game, in order to track the human pose and adapt it onto the game character (Topic: "VR Full Body Tracking"). Very happy about hearing from you!
